index.html

```diff
@@ -1,98 +1,142 @@
-				<ts-button data-toggle="sidebar-inner-left">Toggle Inner Left Sidebar</ts-button>
-				<ts-button data-toggle="sidebar-inner-right">Toggle Inner Right Sidebar</ts-button>
-			</ts-button-group>
-			<script>
-				function toggleSlot(name, button) {
-					var enabledSlot = document.querySelector('[slot=' + name + ']');
-					if (enabledSlot) {
-						button.removeAttribute('type');
-						return enabledSlot.setAttribute('slot', '!' + name);
-					}
+		<!-- Load user-styles -->
+		<link rel="stylesheet" href="/index.css" />
 
-					var disabledSlot = document.querySelector('[slot="!' + name + '"]');
-					if (disabledSlot) {
-						button.setAttribute('type', 'primary');
-						return disabledSlot.setAttribute('slot', name);
-					}
-				}
+		<!-- Load Tradeshift Elements once the polyfills are ready (optional?) -->
+		<script>
+			window.addEventListener('WebComponentsReady', function() {
+				var coreEl = document.createElement('script');
+				coreEl.setAttribute('src', '/packages/core/lib/core.umd.js');
+				document.body.appendChild(coreEl);
+				coreEl.onload = function() {
+					var scripts = [
+						'/packages/components/root/lib/root.umd.js',
+						'/packages/components/button/lib/button.umd.js',
+						'/packages/components/button-group/lib/button-group.umd.js'
+					];
+					scripts.forEach(function(script) {
+						var el = document.createElement('script');
+						el.setAttribute('src', script);
+						document.body.appendChild(el);
+					});
+				};
+			});
+		</script>
+	</head>
+	<body>
+		<ts-root>
+			<header slot="header" class="header">Header</header>
+			<footer slot="footer" class="footer">Footer</footer>
+			<section slot="sidebar-left" class="sidebar-left">Left Sidebar</section>
+			<section slot="sidebar-right" class="sidebar-right">
+				Right Sidebar
+			</section>
+			<section slot="sidebar-inner-left" class="sidebar-inner-left">
+				Inner Left Sidebar
+			</section>
+			<section slot="sidebar-inner-right" class="sidebar-inner-right">
+				Inner Right Sidebar
+			</section>
 
-				(function() {
-					setTimeout(function() {
-						const slots = [
-							'header',
-							'footer',
-							'sidebar-left',
-							'sidebar-right',
-							'sidebar-inner-left',
-							'sidebar-inner-right'
-						];
-						slots.forEach(function(slotName) {
-							var slot = document.querySelector('[slot=' + slotName + ']');
-							if (slot) {
-								slot.setAttribute('slot', '!' + slot.getAttribute('slot'));
-							}
-						});
+			<article id="ts-root-toggle">
+				<h2>ts-root</h2>
+				<ts-button-group>
+					<ts-button onclick="toggleSlot('header', this)"
+						>Toggle Header</ts-button
+					>
+					<ts-button onclick="toggleSlot('footer', this)"
+						>Toggle Footer</ts-button
+					>
+					<ts-button onclick="toggleSlot('sidebar-left', this)"
+						>Toggle Left Sidebar</ts-button
+					>
+					<ts-button onclick="toggleSlot('sidebar-right', this)"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a 'onKeyPress' attribute to this <ts-button> tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMouseEventWithoutKeyboardEquivalentCheck)